tutorials/smart-contracts/launch-your-first-project/index.md

```diff
@@ -1,4 +1,23 @@
+---
```

Comment:
A timeline would be nice on this page. But I think we need to iron out how we're going to roll out the timelines. Because our docs are consistent with the In This Section cards